x-pack/plugins/actions/server/routes/oauth_code.ts

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import qs from 'qs';
+import atob from 'atob';
```

Comment:
I would prefer if we created our own atob function, since it's a one-liner, and doesn't look like we directly include it in our `package.json`; or we should add that package to our `package.json`.  Here's the body from https://git.coolaj86.com/coolaj86/atob.js/src/branch/master/node-atob.js

```js
function atob(str) {
  return Buffer.from(str, 'base64').toString('binary');
}
```